embedchain/loaders/rss_feed.py

```diff
@@ -0,0 +1,49 @@
+import hashlib
+
+try:
+    from langchain.document_loaders import RSSFeedLoader as LangchainRSSFeedLoader
+except ImportError:
+    raise ImportError(
+        'RSSFeedLoader file requires extra dependencies. Install with `pip install --upgrade "embedchain[rss_feed]"`'
+    ) from None
+from embedchain.helper.json_serializable import register_deserializable
+from embedchain.loaders.base_loader import BaseLoader
+
+
+@register_deserializable
+class RSSFeedLoader(BaseLoader):
+    """Loader for RSS Feed."""
+
+    def load_data(self, url):
+        """Load data from a rss feed."""
+        output = self.get_rss_content(url)
+        doc_id = hashlib.sha256((str(output) + url).encode()).hexdigest()
```

Comment:
Should doc_id only contain the URL so that we know if the content changed for the hash or not?

Reply:
This is the right way to know if the content changed or not. If we keep only the url, we will not be able to know the delta.